src/common/mfu_daos.c

```diff
@@ -317,9 +324,11 @@ int daos_parse_path(
 static int daos_set_paths(
   int rank,
   char** argpaths,
+  int numpaths,
   daos_args_t* da)
 {
     int     rc = 0;
+    bool    have_dst = numpaths > 1;
```

Comment:
Style edit for readability:
```suggestion
    bool    have_dst = (numpaths > 1);
```